content/en/docs/Concepts/olm-architecture/operator-catalog/creating-an-update-graph.md

```diff
@@ -255,6 +255,308 @@ entries:
       - myoperator.v0.3.0
 ```
 
+## Skip vs. SkipRange vs. Replaces by Example
+
+For these examples we will be considering an operator that is already installed (`Installed`) and how update options are selected for it.
+
+```mermaid
+graph LR
+  subgraph Update<br>Options
+    a
+    b
+    c
+  end
+  I(Installed) 
+  a -->|can update from| I
+  b -->|can update from| I
+  c -->|can update from| I
+```
+
+Invariants to keep in mind:
+
+- Anything in the channel that `replaces` the `Installed` package is an option.
+- Anything in the channel that is `skipped` by anything else in the channel is **not** considered an option.
+- Anything in the channel that has a `skipRange` that includes `Installed` is an option.
+
+Once there's a set of options found, they're "tried" by the resolver in order. This try-order in is defined by their `depth` in the channel **as defined by replaces and skips only**. 
+
+### Incremental update validation
+
+When a new version is released, it is tested against the latest release in the channel.
```

Comment:
Are lines like these relevant in our upstream docs? I'm not sure which tests you're talking about here.